src/main/java/com/google/sps/data/Grocery.java

```diff
@@ -23,13 +23,15 @@
   private final long id;
   private final String email;
   private final boolean userMatch;
+  private final Boolean complete;
 
-  public Grocery(String email, long id, long timestamp, String item, boolean userMatch) {
+  public Grocery(String email, long id, long timestamp, String item, boolean userMatch, Boolean complete) {
```

Comment:
Use a consistent bool type - `boolean`.

Reply:
I needed to use Boolean because I wouldn't let me get the property using boolean.

Reply:
I didn't quite understand. Why does userMatch work even though that is a `boolean`?

Reply:
In my groceryServlet I have to get the property of a boolean from datastore for complete. The only way to do that is with Boolean. Usermatch isn't getting any booleans from the datastore. It just grabs the assigned users name and returns true or false based off of that,